core/platform/email/mailgun_email_services.py

```diff
@@ -16,10 +16,32 @@
 
 """Provides mailgun api to send email."""
 
+import base64
+import urllib
+import urllib2
 from core.platform.email import gae_email_services
 import feconf
 
-import requests
+
+def post_to_mailgun(data):
+    """Send POST HTTP request to mailgun api. This method is adopted from
+    `requests`'s post method.
```

Comment:
Sorry, let's make this "the requests library's post method". (Backticks don't really translate well here.)

Reply:
Done